services/brig/test/integration/Federation/End2end.hs

```diff
@@ -592,7 +592,15 @@ claimRemoteKeyPackages brig1 brig2 = do
     Set.map (\e -> (e.user, e.client)) bundle.entries
       @?= Set.fromList [(bob, c) | c <- bobClients]
 
-testRemoteTypingIndicator :: Brig -> Brig -> Galley -> Galley -> Cannon -> Cannon -> Http ()
+testRemoteTypingIndicator ::
```

Comment:
This seems unrelated to this PR.

Reply:
This is a left-over from debugging this test as it was initially failing.

I tend to split the screen vertically and then these long signatures don't fit into a half of the screen, and it gets annoying having to move the cursor/slider to see what are the rest of the arguments.

If you insist, I'll remove it from this PR, but I'd argue the change makes the code more ergonomic.

Reply:
All good.